Jenkinsfile

```diff
@@ -1,8 +1,16 @@
 @Library('jenkins-library') _
 
+if (env.GIT_BRANCH == "master" || env.GIT_BRANCH == "develop") {
+    buildEnvironment = ['NODE_ENV': 'test']
+    echo "buildEnvironment ${buildEnvironment}"
```

Comment:
let's remove echo

Reply:
sure, just for debug and forgot removing